### PR TITLE
Update community-membership.md

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Linkify openprofile.dev.

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: community-membership.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ remain active contributors to the community.
     - Contributing to SIG, subproject, or community discussions (e.g. meetings,
       Slack, email discussion forums)
 - Subscribed to [dev@kubernetes.io]
-- Have read the [contributor guide]
+- Have read the [contributor guide]()
```
Comment (ArvindParekh) [inline community-membership.md:64]: Also, this is not necessary :)

**Review by ArvindParekh — COMMENTED** [review]
Hi @benjaminapetersen, thanks for the PR! However, this might not be the best approach — since we're referencing all the links together at the last. Also, there are already two PRs currently under review that address this particular change (see [#8117](https://github.com/kubernetes/community/pull/8117) and [#8028](https://github.com/kubernetes/community/pull/8028)).

You might want to reference these PRs to see the overlap. Given that, I suggest we close this PR to avoid duplication.
/hold

File: community-membership.md (hunk 1 above)
Comment (benjaminapetersen) [reply community-membership.md:64]: I see that now, my bad.  Thanks @ArvindParekh!

**Comment by benjaminapetersen** [comment]
Agree, thanks!

**Pull request closed without merging**